src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {

```

Comment:
Extending the `ColorSupplier` class in `Lottery` is not necessary. It's better to use composition over inheritance in this case. You can create an instance of `ColorSupplier` inside the `Lottery` class and use it to get the random color.